R/convGDX2MIF.R

```diff
@@ -95,12 +95,24 @@ convGDX2MIF <- function(gdx,gdx_ref=NULL,file=NULL,scenario="default",t=c(seq(20
       warning(paste0("File ",gdx_ref," not found. Did not execute 'reportPolicyCosts'! If a policy costs reporting is desired, please use the   'policyCosts' output.R script."))
     }
   }
+  
+  # reporting of SDP variables
+  message("running reportSDPVariables...")
+  tmp <- try(reportSDPVariables(output))  # test whether reportSDPVariables works
+  if(class(tmp)!="try-error") {
+    if(!is.null(tmp)) output <- tmp
+  } else {
+    message("function reportSDPVariables does not work and is skipped")
+  }
+  
```

Comment:
do I get it right: the output is not added but overwritten?

Reply:
I would prefer calculating the new variables in your new function an adding it to the existing output instead of doing the adding in each single sub-function